smart-leds/src/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "robotka.h"
+
+#define LED_COUNT 8
+
+void showGradient() {
+    static uint8_t hue;
+    hue++;
```

Comment:
+1 je hrozně pomalu, až to není skoro vidět, že se mění. Dej klidně +10 nebo +20.